db/migrations/20191112111930_backfill_movies_name.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+exports.up = (knex) => {
+  return knex.raw('UPDATE movies SET name = title');
```

Comment:
you need to add the name column first right?

Reply:
Yes, I have already added the name column in a previous migration.